doc-content/enterprise-only/openshift/ldap_rhsso.adoc

```diff
@@ -0,0 +1,95 @@
+// This include file requires the following attribute settings:
+// * one of kieonly, centralonly, or kieandcentral
+// * optional multiplekie (for the managed templates)
+// * LOCAL_CENTRAL must be set to the name of Business Central, Business Central Monitoring, Decision Central
+// It turns out that asciidoc does not alow attribute setting in the middle of a list, so these attributes must be set at 
+// the beginning of the parent file.
+. If you want to use RH-SSO or LDAP authentication, complete the following additional configuration:
+.. In the RH-SSO or LDAP service, create all user names in the deployment parameters. If you do not set any of the parameters, create the default user names:
```

Comment:
The multi-nested steps/parts of this section is I see somewhat unavoidable, under the circumstances. But in general, what I find helpful to minimize overly nested or overly complex step structures is to instead of saying what users can do ("If you want to use RH-SSO.."), I just pick the recommended path and then instruct them to do it as part of the required steps and why. And I find most of the time that my nested steps can actually be parent-level steps or non-numbered steps (usually anything that starts with "If" is not a step but an non-numbered option of a parent step). If truly necessary, I'll put in parentheses next to the relevant step or in a note or whatever mention that this step is optional or if they can skip the step in X case. So here, I would just build in RH-SSO steps as part of the steps and simplify nesting where possible. For the sake of time, you can ignore this comment and move forward as is, but for future reference, just a thought.